jsonpickle/pickler.py

```diff
@@ -234,6 +234,7 @@ def __init__(
         self._flattened = {}
         # Used for util.is_readonly, see +483
         self.handle_readonly = handle_readonly
+        self.excludes = []
```

Comment:
`self.excludes = set()`

Reply:
I support this. Shouldn't it be able to be specified at creation time too though, like through the encode interface?

Reply:
Moved to use set - much better.

If you specify in the encode interface you are back to the broad brush approach, but should be easily doable.